sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/SdkMeterProviderTest.java

```diff
@@ -554,9 +554,11 @@ void viewSdk_AllowMulitpleViewsPerAsynchronousInstrument() {
 
   @Test
   @SuppressWarnings("unchecked")
-  void collectAllAsyncInstruments_CumulativeCount() {
```

Comment:
did we intentionally lose the tests for counts?

Reply:
count aggregator isn't exposed publicly.  Instead using a histogram to do counts, yes.